src/util/util.js

```diff
@@ -89,7 +90,8 @@ export function renderGraphiQL({ query, variables, version = GRAPHIQL_VERSION }
               method: 'post',
               headers: {
                 'Accept': 'application/json',
-                'Content-Type': 'application/json',
+                'Content-Type': 'application/json'
```

Comment:
@CMTegner good points. I don't have time to see this through to completion, as I no longer use this library.